tools/setup/setup.sh

```diff
@@ -220,36 +178,34 @@ fi
 if [[ "$MODE" == "stable" || ! -v MODE ]]; then
 # Stable mode
     if [[ $DEVICE == "tpu" ]]; then
-        # TODO: Once tunix has support for GPUs, move it from here to requirements.txt
-        echo "Installing google-tunix for stable TPU environment"
-        python3 -m uv pip install 'google-tunix>=0.1.2'
         echo "Installing stable jax, jaxlib for tpu"
         if [[ -n "$JAX_VERSION" ]]; then
             echo "Installing stable jax, jaxlib, libtpu version ${JAX_VERSION}"
-            python3 -m uv pip install -U jax[tpu]==${JAX_VERSION} -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
+            python3 -m uv pip install -U jax[tpu]=="${JAX_VERSION}" \
+                -f 'https://storage.googleapis.com/jax-releases/libtpu_releases.html'
         fi
         if [[ -n "$LIBTPU_GCS_PATH" ]]; then
             # Install custom libtpu
             echo "Installing libtpu.so from $LIBTPU_GCS_PATH to $libtpu_path"
             # Install required dependency
-            python3 -m uv pip install -U crcmod
+            python3 -m uv pip install -U 'crcmod'
```

Comment:
Should this be with `--no-deps`?

Reply:
Guarded by `if [[ -n "$LIBTPU_GCS_PATH" ]]` I assume; as it's not needed in other branches